sdk/storage/azure-storage-file-share/README.md

```diff
@@ -24,7 +24,7 @@ Shares provide a way to organize sets of files and also can be mounted as an SMB
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-share</artifactId>
```

Comment:
Preview.5 used artifactId azure-storage-file

Reply:
Also, it wasn't preview.5 it was preview.4, I think this was double bumped at some point

Reply:
Scratch these, I see the other artifact. Approved.

Reply:
Just to close out this conversation. 12.0.0-preview.5 is the correct version for [azure-storage-file-share](https://search.maven.org/search?q=a:azure-storage-file-share)